packages/components/src/internal/util/WindowFocusCheckExpiredSession.tsx

```diff
@@ -0,0 +1,32 @@
+import React, { FC, useEffect } from 'react';
+
+import { useServerContext } from '../components/base/ServerContext';
+
+export const WindowFocusCheckExpiredSession: FC = () => {
```

Comment:
nit: Consider using the hook-paradigm instead -- something like `useWindowFocusCheckExpiredSession` and then having that be a hook utilized in the apps (sorry didn't look at the other PRs and how this is wired in).

Reply:
Done. Ready for re-review